spec.html

```diff
@@ -18452,9 +18451,8 @@ <h1>
       </dl>
       <emu-alg>
         1. Assert: _identifierName_ is an |IdentifierName|.
-        1. Let _bv_ be ? RequireObjectCoercible(_baseValue_).
```

Comment:
By removing the call to RequireObjectCoercible, it appears that EvaluatePropertyAccessWithIdentifierKey can no longer return an abrupt completion, so invocations of it should be prefixed with `!` rather than `?`.

Reply:
Nice catch, done.